src/stats.bif

```diff
@@ -484,3 +484,33 @@ function get_reporter_stats%(%): ReporterStats
 
 	return r;
 	%}
+
+## Returns a list of event handlers that were called and the number of times
+## each was called.
+##
+## Returns: A record with event call statistics.
+##
+function get_event_handler_call_counts%(%): EventNameStats
+  %{
+	auto rval = zeek::make_intrusive<zeek::VectorVal>(zeek::id::find_type<VectorType>("EventNameStats"));
+	const auto& recordType = zeek::id::find_type<RecordType>("EventNameCounter");
+
+	int i = 0;
+	const auto& events = event_registry->UsedHandlers();
+	for ( const auto& name : events )
+		{
+		auto handler = event_registry->Lookup(name);
+		auto call_count = handler->CallCount();
+
+		if ( call_count > 0 )
+			{
+			auto eventStatRecord = zeek::make_intrusive<zeek::RecordVal>(recordType);
+			eventStatRecord->Assign(0, zeek::make_intrusive<zeek::StringVal>(name));
+			eventStatRecord->Assign(1, zeek::val_mgr->Count(handler->CallCount()));
+			rval->Assign(i, std::move(eventStatRecord));
```

Comment:
For the future: VectorVal has an `->Append()` which looks a bit nicer.

Reply:
Fixed in c0bd2e43b120bfff490424d4ac54c669c7024d96, along with your other comments below.